RZBluetooth/RZBPeripheral+Private.h

```diff
@@ -0,0 +1,28 @@
+//
+//  RZBPeripheral+Private.h
+//  RZBluetooth
+//
+//  Created by Brian King on 3/22/16.
+//  Copyright © 2016 Raizlabs. All rights reserved.
+//
+
+#import "RZBPeripheral.h"
+#import "CBService+RZBExtension.h"
+#import "RZBCentralManager+Private.h"
+#import "RZBUUIDPath.h"
+#import "RZBCommand.h"
+#import "RZBDefines.h"
+
+@interface RZBPeripheral ()
+
+@property (weak, nonatomic, readonly) RZBCentralManager *centralManager;
+
+@property (strong, nonatomic, readonly) CBPeripheral *corePeripheral;
```

Comment:
#? This too. If we are going to go to the `RZBPeripheral` we should only allow the use of them and no `CBPeripheral`